user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/storage_per_DMA/Profile_migration_to_elastic.md

```diff
@@ -0,0 +1,103 @@
+
+   - All profiles will be loaded from the *Profiles.xml* file on the DataMiner Agent where the migration was triggered. The *Profiles.xml* files of the other DataMiner Agents in the cluster are not migrated. This is not necessary, as the *Profiles.xml* file is synced in the cluster.
+
+   - The profiles objects will be written to the indexing database.
+
+   - When the migration for profiles is completed, the configuration will not switch to indexing database storage. To switch to indexing database storage, press the *Start using Elastic* button.
+
+> [!NOTE]
+>
+> - The migration should not take more than a few minutes. During testing, migrating a *Profiles.xml* file of 100 MB in a system with a local Elasticsearch database took about 1 minute.
+> - You can cancel the migration process by clicking the *Cancel Migration* button in the *ProfileManager XML to Elastic* section.
+
+### Troubleshooting
+
+If the migration should fail for any reason, the migration status object in the SLNetClientTest tool window will get a red background color. The ``SLMigrationManager.txt`` and ``SLProfileManager.txt`` log files will contain more information.
+
+If a ``MigrationStatus`` is stuck in the ``InProgress`` state, you will need to cancel the migration and trigger the migration again. You can do so with the *Cancel Migration* button in the *ProfileManger XML to Elastic* section of the SLNetClientTest tool window.
+
+### Behavior in new installations
+
+When a new DataMiner Agent is installed, the used storage type will depend on when Profile Manager starts up for the first time.
+
+- If DataMiner is installed with the 10.0 installer and Profile Manager is used, XML storage will be used. An indexing database is not yet supported as a storage type for profiles in DataMiner 10.0. After you have upgraded this DataMiner Agent to DataMiner 10.0.8 or later, it will continue to use XML storage until you trigger the migration.
+
+- If DataMiner is installed on version 10.4.0 and Profile Manager is used, indexing database storage will be used. This means profiles will only be functional on a DataMiner System that has indexing database storage running.
+
+- When you add a new DataMiner Agent to an existing cluster, the same behavior applies as mentioned above.
+
+## Checking the storage type used by a DataMiner Agent
+
+If you want to know which storage type a DataMiner Agent is currently using, open the migration window as detailed under [Migrating from XML to the indexing database](#migrating-from-xml-to-the-indexing-database).
+
+The current storage type is displayed in the *ProfileManger XML to Elastic* section, under the *Cancel Migration* button stating "Currently using ..." .
+
+## Limitations and differences with XML storage
+
+### Difference when deleting profiles
```

Comment:
You can delete this difference, profiles objects never had a restriction for deletes (not on Xml or on Elastic storage)